docs/chapters/przyjazne_si.tex

```diff
@@ -0,0 +1,27 @@
+\subsection{Sztuczna inteligencja przyjaznych jednostek.}
+
+Kontrola jednostek przez gracza odbywa się za pomocą wybory jednej z opcji w dwóch fazach
+\begin{itemize}
+\item Faza wybrania jednostek
+  \begin{itemize}
+    \item Wszyscy
+    \item Jednostki bliskozasięgowe 
+    \item Jednostki hybrydowe
+    \item Jednostki dalekozasięgowe
+    \item Opcja anulowania wyboru
+  \end{itemize}
+
+\item Faza wydania rozkazu
+  \begin{itemize}
+    \item Podążanie za graczem
+    \item Zatrzymanie
+    \item Atak
+    \item Podejście do wskazanego przez gracza miejsca
+    \item Ucieczki
+      \item Opcja anulowania rozkazu
+  \end{itemize}
+\end{itemize}
+Wydanie rozkazu polega na kliknięciu przycisku odpowiedzialnego za wejście w tryb wydawania poleceń, a następnie
+wybraniu numeru opcji reprezentującą grupę jednostek, której komenda ma dotyczyć. Ostatecznie należy podać numer rozkazu, który zostanie wydany.
+Przykładowa reprezentacji graficzna systemu jest widoczna w rysunku \ref{fig:mnb}.
```

Comment:
I'n not 100% sure but maybe: "Wydanie rozkazu polega na kliknięciu przycisku odpowiedzialnego za wejście w tryb wydawania poleceń, a następnie
wybraniu numeru opcji reprezentują**cej** grupę jednostek, której komenda ma dotyczyć. Ostatecznie należy podać numer rozkazu, który zostanie wydany.
Przykładowa reprezentacj**a** graficzna systemu jest widoczna **na** rysunku"